app/src/main/java/com/amaze/filemanager/filesystem/FileUtil.java

```diff
@@ -91,13 +94,18 @@
   private static final Pattern FILENAME_REGEX =
       Pattern.compile("[\\\\\\/:\\*\\?\"<>\\|\\x01-\\x1F\\x7F]", Pattern.CASE_INSENSITIVE);
 
-  /**
-   * Determine the camera folder. There seems to be no Android API to work for real devices, so this
-   * is a best guess.
-   *
-   * @return the default camera folder.
-   */
-  // TODO the function?
+  private static final String EXTERNAL_STORAGE_PREFIX;
+
+  private static final List<String> EXCLUDED_DIRS =
+      Arrays.asList(
+          new File(Environment.getExternalStorageDirectory(), "Android/data").getAbsolutePath(),
+          new File(Environment.getExternalStorageDirectory(), "Android/obb").getAbsolutePath());
+
+  static {
```

Comment:
Can we use `DataUtils.getStorages` here? 🤔

Reply:
If the first path returned by `DataUtils.getStorages()` is guaranteed to be the path for the equivalent of `Environment.getExternalStorageDirectory()`, yes.

Reply:
I really think using static block isn't ideal. It'll slow down application load times. 
Please see if you can avoid it. `DataUtils.getStorages` does have first path equivalent to external storage most of the times, maybe can be tested on few devices.